packages/venia-ui/lib/components/Header/header.css

```diff
@@ -61,7 +61,7 @@
     .toolbar {
         grid-template-areas: 'primary title secondary';
         grid-template-columns: auto auto 1fr;
-        min-height: 5rem;
+        min-height: var(--venia-global-header-minHeight);
```

Comment:
Should we switch this variable to `rem`? I believe it uses `px` after your change, but I didn't realize it was going to be used here.

Reply:
All values in the tokens.css were px so I converted these to px as well. But after discussing with the team, I realized there is no standard that we follow at the moment for tokens so changed it back to `rem`. Nice question @sirugh 😄